solution/zenkoversion.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: 1.1-dev
```

Comment:
this would need to be `${PRODUCT_LOWERNAME}-${VERSION_FULL}` ? the solution config seems to be using that.

Reply:
I think it just needs to match the "version" value from the Solution manifest. Odd how one is called version while the other is name though.